src/components/faceRecognition/FaceRecognition.js

```diff
@@ -1,20 +1,20 @@
-import React from 'react';
 import './FaceRecognition.css';
 
-const FaceRecognition = ({ imageUrl, box }) => {
-    // box.map = (boxes) => {
-        
-    // }
+const FaceRecognition = ({ imageUrl, boxes }) => {
+
     return(
         <div className='center ma'>
             <div className='absolute mt2'>
                 <img id='inputImage' alt='face for processing' src={ imageUrl } width='500px' height='auto' />
-                <div className='boundingBox'style={{ top:box.topRow, left:box.leftCol, right:box.rightCol, bottom:box.bottomRow }}>
-                </div>
-                
+                    {
+                        boxes.map((box, id) => (
+                            <div key={id} className='boundingBox'style={{ top:box.topRow, left:box.leftCol, right:box.rightCol, bottom:box.bottomRow }}></div>
```

Comment:
Extract to own component
```suggestion
                            <FaceBox />
```